From bffd8df1725c0fae9105a097e75b466e2f49d7bd Mon Sep 17 00:00:00 2001
From: "Enrico Weigelt, metux IT consult" <info@metux.net>
Date: Fri, 6 Sep 2024 13:53:10 +0200
Subject: [PATCH] .gitmodules: fix xcb-util-m4 url and CI failure

anongit.freedesktop.org doesn't seem to work anymore - using the canonical
gitlab https URL instead, so making the CIs work again.

Signed-off-by: Enrico Weigelt, metux IT consult <info@metux.net>
Part-of: <https://gitlab.freedesktop.org/xorg/lib/libxcb-util/-/merge_requests/7>
---
 .gitmodules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.gitmodules b/.gitmodules
index cff1b86..375bb2e 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,3 +1,3 @@
 [submodule "m4"]
 	path = m4
-	url = git://anongit.freedesktop.org/xcb/util-common-m4.git
+	url = https://gitlab.freedesktop.org/xorg/util/xcb-util-m4.git
-- 
2.43.0

